### PR TITLE
Added github oauth support(WebAPI #709)

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes https://github.com/OHDSI/WebAPI/issues/709

### CODE REVIEW

File: js/Application.js [hunk 1]
```diff
@@ -242,7 +242,14 @@ define(
 					}
 				});
 			}
-
+      checkOAuthError() {
```
Comment (pavgra) [inline js/Application.js]: indentation

File: js/Application.js [hunk 2]
```diff
@@ -242,7 +242,14 @@ define(
...
+        let hash = window.location.hash;
+        if (hash) {
+          if (hash.includes("oauth_error_email")) {
```
Comment (pavgra) [inline js/Application.js]: `hash && hash.includes("oauth_error_email")` instead of nested `if`

File: js/Application.js (hunk 1 above)
Comment (ssuvorov-fls) [reply js/Application.js]: fixed

File: js/Application.js (hunk 2 above)
Comment (ssuvorov-fls) [reply js/Application.js]: fixed

**Pull request merged**